lib/output.js

```diff
@@ -632,7 +632,7 @@ function OutputStream(options) {
     /* -----[ PRINTERS ]----- */
 
     DEFPRINT(AST_Directive, function(self, output){
-        output.print_string(self.value, self.quote);
+        output.print(self.quote + self.value + self.quote);
```

Comment:
`print_string()` respects `-b quote_style=[0123]`. This change does not.

Also not sure about whether this new way would handle mis-shaped directives with escaped characters correctly.